src/app/container/deregister-modal/deregister-modal.component.html

```diff
@@ -0,0 +1,21 @@
+<button id="deregisterButton" type="button" class="btn btn-danger" (click)="staticModal.show()">
+  Deregister
+</button>
+<div class="modal fade" bsModal #staticModal="bs-modal" [config]="{backdrop: 'static'}" tabindex="-1" role="dialog" aria-labelledby="mySmallModalLabel" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <div class="modal-title">Are you sure you wish to deregister this tool?</div>
+      </div>
+      <div class="modal-body">
+        <p>All information associated with this tool will be deleted.</p>
```

Comment:
Idle thought: Should this be called "Delete" rather than "Deregister"? That may require less explanation although it would parallel "register new tool" less.